packages/use-storage/src/index.ts

```diff
@@ -17,7 +17,6 @@ declare global {
 }
 
 const canUseDOM = !!(
-  typeof window !== 'undefined' &&
```

Comment:
why ? This is going to generate an exception the line after if window is undefined :)

Reply:
yeah i know, it's eslint fix that change that.